### PR TITLE
Translate "Hacking" (#7)

### PR DESCRIPTION
**Pull request by danya02**

### CODE REVIEW

File: translation.rst [hunk 1]
```diff
@@ -122,10 +122,10 @@ If I were in college now I'd probably work on graphics: a network game, for exam
 Notes
 -----
 
-.. [1] No one seems to have minded, which shows how unimportant the Arpanet (which became the Internet) was as late as 1984.
+.. [1] Никому до этого не было дела, что показывает, насколько Arpanet (ставший Интернетом) был в 1984 году.
```
Comment (wldhx) [inline translation.rst]: Насколько _что_ был?

File: translation.rst [hunk 2]
```diff
@@ -122,10 +122,10 @@ If I were in college now I'd probably work on graphics: a network game, for exam
...
 
 .. [2]
-        This is why, when I became an employer, I didn't care about GPAs. In fact, we actively sought out people who'd failed out of school. We once put up posters around Harvard saying "Did you just get kicked out for doing badly in your classes because you spent all your time working on some project of your own? Come work for us!" We managed to find a kid who had been, and he was a great hacker.
+        Поэтому, став работодателем, мне не были важны оценки. Мы даже активно искалиwe actively sought out people who'd failed out of school. We once put up posters around Harvard saying "Did you just get kicked out for doing badly in your classes because you spent all your time working on some project of your own? Come work for us!" We managed to find a kid who had been, and he was a great hacker.
```
Comment (wldhx) [inline translation.rst]: Ow.

**Comment by danya02** [comment]
Ouch. 

I just didn't notice that part. Will fix a bit later.

**Comment by danya02** [comment]
Fixed those.

File: translation.rst [hunk 3]
```diff
@@ -122,12 +122,12 @@ If I were in college now I'd probably work on graphics: a network game, for exam
 Notes
 -----
 
-.. [1] No one seems to have minded, which shows how unimportant the Arpanet (which became the Internet) was as late as 1984.
+.. [1] Никому до этого не было дела, что показывает, насколько Arpanet (ставший Интернетом) был маловажным в 1984 году.
 
 .. [2]
-        This is why, when I became an employer, I didn't care about GPAs. In fact, we actively sought out people who'd failed out of school. We once put up posters around Harvard saying "Did you just get kicked out for doing badly in your classes because you spent all your time working on some project of your own? Come work for us!" We managed to find a kid who had been, and he was a great hacker.
+        Поэтому, став работодателем, мне не были важны оценки. Мы даже активно искали людей, проваливших школу. Мы однажды повесили по Гарварду плакаты, на которых было написано: "Вас выкинули за плохую успеваемость из-за того, что были слишком заняты своим проектом? Приходите к нам на работу!" Мы нашли одного такого, и он был отличным хакером.
```
Comment (wldhx) [inline translation.rst]: > Поэтому, став работодателем, мне не были важны оценки.

"Проезжая мимо станции, с меня слетела шляпа."

File: translation.rst [hunk 4]
```diff
@@ -7,30 +7,30 @@ March 2005
...
+В собственных проектах не надо задумываться об оригинальности (как приходится профессорам) и о рентабельности (как приходится компаниям). Важна лишь техническая сложность, а она никак не связана с природой применения. "Серьезные" программы, к примеру, базы данных, часто тривиальны и технически скучны (если страдаете от бессоницы, почитайте техническую литературу про базы данных), а "фривольные" программы, как игры, часто очень сложные. Уверен, что существуют игровые компании, работающие над проектами с большей частью интеллектуального содержимого, чем в 9/10 исследований университетских факультетов информатики.
 
-If I were in college now I'd probably work on graphics: a network game, for example, or a tool for 3D animation. When I was an undergrad there weren't enough cycles around to make graphics interesting, but it's hard to imagine anything more fun to work on now.
+Будь я сейчас в колледже, я бы работал над графикой: к примеру, над сетевой игрой, или программой 3D-анимации. Будучи студентом, не было достаточного количества вычислительных мощностей, чтобы графика была интересной, но сейчас сложно найти что-либо более увлекательное.
```
Comment (wldhx) [inline translation.rst]: Причастный оборот: "будучи студентом ...".

**Comment by wldhx** [comment]
There are a few grammar errors I've highlighted; otherwise, seems ok.

**Comment by danya02** [comment]
Fixed those. BTW, what kind of linter did you use? The default (vgrep) doesn't seem to do the trick.

**Comment by danya02** [comment]
No, missed one. Fixing in progress.

**Comment by wldhx** [comment]
> vgrep

If you are referring to "visual grep", precisely that. Not sure if we have any really good automated natural language (and Russian, for that matter) linters.

**Comment by danya02** [comment]
Fixed that one (which was on line 128).

**Comment by wldhx** [comment]
@danya02 Nice :+1:! There's only metatext left now; I think I'll translate it myself.

**Comment by danya02** [comment]
@wldhx Great news! Maybe we have a chance of completing the translation after all!

**Pull request merged**